Container/src/V1/Gapic/ClusterManagerGapicClient.php

```diff
@@ -2396,4 +2400,95 @@ public function setMaintenancePolicy($projectId, $zone, $clusterId, $maintenance
             $request
         )->wait();
     }
+
+    /**
+     * Lists subnetworks that are usable for creating clusters in a project.
+     *
+     * Sample code:
+     * ```
+     * $clusterManagerClient = new Google\Cloud\Container\V1\ClusterManagerClient();
+     * try {
+     *     // Iterate over pages of elements
+     *     $pagedResponse = $clusterManagerClient->listUsableSubnetworks();
+     *     foreach ($pagedResponse->iteratePages() as $page) {
+     *         foreach ($page as $element) {
+     *             // doSomethingWith($element);
+     *         }
+     *     }
+     *
+     *
+     *     // Alternatively:
+     *
+     *     // Iterate through all elements
+     *     $pagedResponse = $clusterManagerClient->listUsableSubnetworks();
+     *     foreach ($pagedResponse->iterateAllElements() as $element) {
+     *         // doSomethingWith($element);
+     *     }
+     * } finally {
+     *     $clusterManagerClient->close();
+     * }
+     * ```
+     *
+     * @param array $optionalArgs {
+     *                            Optional.
+     *
+     *     @type string $parent
+     *          The parent project where subnetworks are usable.
+     *          Specified in the format 'projects/*'.
+     *     @type string $filter
+     *          Filtering currently only supports equality on the networkProjectId and must
+     *          be in the form: "networkProjectId=[PROJECTID]", where `networkProjectId`
+     *          is the project which owns the listed subnetworks. This defaults to the
+     *          parent project ID.
+     *     @type int $pageSize
+     *          The maximum number of resources contained in the underlying API
+     *          response. The API may return fewer values in a page, even if
+     *          there are additional values to be retrieved.
+     *     @type string $pageToken
+     *          A page token is used to specify a page of values to be returned.
+     *          If no page token is specified (the default), the first page
+     *          of values will be returned. Any page token used here must have
+     *          been generated by a previous call to the API.
+     *     @type RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     * }
+     *
+     * @return \Google\ApiCore\PagedListResponse
+     *
+     * @throws ApiException if the remote call fails
+     * @experimental
+     */
+    public function listUsableSubnetworks(array $optionalArgs = [])
```

Comment:
should confirm that $parent is not required.

Reply:
I'll confirm upstream if this is intended.